source/guides/clear/clear-linux-dissector.rst

```diff
@@ -0,0 +1,302 @@
+.. _clear-linux-dissector:
+
+Clear Linux\* Dissector
+#######################
+
+The Clear Linux\* Dissector tool compares contents of Linux-based images. It
+supports the comparison of |CL| and Yocto Project\* images.
+
+.. contents:: :local:
+   :depth: 1
+
+Description
+***********
+
+Clear Linux Dissector provides tools for comparing |CL| images and Yocto Project
+images.
```

Comment:
There are some technicalities here - at the moment it can compare a Yocto Project image with the entire Clear Linux distribution (i.e. with a view to having a hypothetical Clear Linux image that contains all equivalent components). You can also search for a specific component in Clear without reference to an image and see how it compares to its generic YP equivalent.